charts/prometheus-nginx-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: v2
 description: A Helm chart for NGINX Prometheus Exporter
 name: prometheus-nginx-exporter
-version: 1.7.1
+version: 1.7.2
```

This change adds a new feature, is minor:
```suggestion
version: 1.8.0
```